build/Rulesets/NonShippingProjectRules.ruleset

```diff
@@ -15,4 +15,9 @@
     <Rule Id="RS0006" Action="None" />
     <Rule Id="RS0023" Action="None" />
   </Rules>
+   <Rules AnalyzerId="Microsoft.CodeAnalysis.CSharp.Features" RuleNamespace="Microsoft.CodeAnalysis.CSharp.Features">
+     <!-- Naming styles is too noisy as it fires on all async tests -->
```

Comment:
NonShippingProjectRules is effectively that, it applies to all non-shipping projects, which in our tree is the majority of the test projects.